src/Symfony/Component/Notifier/Bridge/Matrix/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "symfony/matrix-notifier",
+  "type": "symfony-notifier-bridge",
+  "description": "Symfony Matrix Synapse Notifier Bridge",
+  "keywords": ["matrix", "notifier"],
+  "homepage": "https://symfony.com",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Frank Schulze",
+      "email": "frank@akiber.de"
+    },
+    {
+      "name": "Symfony Community",
+      "homepage": "https://symfony.com/contributors"
+    }
+  ],
+  "require": {
+    "php": ">=8.2",
+    "symfony/notifier": "^7.3",
+    "symfony/uid": "^7.2",
```

Comment:
Do we need this dependency?

Reply:
Looks like it is.

Reply:
Ah the file was collapsed in my Github view, that's why I didn't see it